mlflow/genai/labeling/__init__.py

```diff
@@ -8,18 +8,30 @@
 
 from typing import Any, Optional
 
-try:
-    from databricks.agents.review_app import (
-        Agent,
-        LabelingSession,
-        ReviewApp,
-        get_review_app,
-    )
-except ImportError:
-    raise ImportError(
-        "The `databricks-agents` package is required to use `mlflow.genai.labeling`. "
-        "Please install it with `pip install databricks-agents`."
-    ) from None
+from mlflow.genai.labeling.labeling import Agent, LabelingSession, ReviewApp
+
+_ERROR_MSG = (
+    "The `databricks-agents` package is required to use `mlflow.genai.labeling`. "
+    "Please install it with `pip install databricks-agents`."
+)
+
+
+def get_review_app(experiment_id: Optional[str] = None) -> "ReviewApp":
+    """Gets or creates (if it doesn't exist) the review app for the given experiment ID.
+
+    Args:
+        experiment_id: Optional. The experiment ID for which to get the review app. If not provided,
+            the experiment ID is inferred from the current active environment.
+
+    Returns:
+        ReviewApp: The review app.
+    """
+    try:
+        from databricks.agents.review_app import get_review_app as _get_review_app
+    except ImportError:
+        raise ImportError(_ERROR_MSG) from None
+
+    return _get_review_app(experiment_id)
```

Comment:
do we need to wrap the agents.ReviewApp into an mlflow.genai.ReviewApp ?

Reply:
Yup good catch, fixed this, below comment, and in other places I noticed.